include/benchmark/benchmark.h

```diff
@@ -1243,8 +1243,15 @@ internal::Benchmark* RegisterBenchmark(const char* name, Lambda&& fn) {
 template <class Lambda, class... Args>
 internal::Benchmark* RegisterBenchmark(const char* name, Lambda&& fn,
                                        Args&&... args) {
+#if defined(BENCHMARK_HAS_CXX14)
```

Comment:
Where is this macro defined?

Reply:
Please just see where in this header the `BENCHMARK_HAS_CXX11` is defined, and do the same for `BENCHMARK_HAS_CXX14`.